### ISSUE TITLE
(🐞) F722 false positive 

### ISSUE THREAD

**Issue by KotlinIsland** [astral-sh/ruff#6111]
```py
from __future__ import annotations

from typing import Literal

E = Literal["01"]  #  F722 Syntax error in forward annotation: `01`
```

Python 3.11
ruff ruff 0.0.280

**Comment by charliermarsh** [comment]
This is fixed on main, sorry about that.
